pages/linux/sacct.md

```diff
@@ -0,0 +1,12 @@
+# sacct
+
+> Display accounting data from the Slurm service.
+> More information: <https://slurm.schedmd.com/sacct.html>.
+
```

Comment:
Could you add the example for just `sacct` without params and `sacct --brief` (looks useful to see what is running)?